react-common/components/controls/Link.tsx

```diff
@@ -2,8 +2,9 @@ import * as React from "react";
 import { classList, ContainerProps } from "../util";
 
 export interface LinkProps extends ContainerProps {
-    href: string;
+    href?: string;
     target?: "_self" | "_blank" | "_parent" | "_top";
+    onClick?: () => void;
```

Comment:
instead of this, use the link-button class on the Button component. better for accessibility